handbook/product/README.md

```diff
@@ -381,8 +381,11 @@ Directly Responsible Individuals (DRI) engage in the ritual(s) below at the freq
 | Product development process review | Sprintly | CEO, Director of Product Development, and Head of Product get together to review boards and process to make sure everything still makes sense | Mo Zhu |
 | Report number of estimated stories (CX) | Weekly | Report the number of product-driven stories estimated that week to the Head of Product for KPI tracking purposes | Mo Zhu |
 | Report number of estimated stories (MDM) | Weekly | Report the number of product-driven stories estimated that week to the Head of Product for KPI tracking purposes | Noah Talerman |
-
-| Bug de-prioritization | Sprintly  | CEO and Head of Product review the bugs which did not meet the 6-week delivery timeline. Align on next steps for which bugs to continue into the next sprint and which can be de-prioritized. | Mo Zhu |
+| Bug de-prioritization | Sprintly  | CEO, Head of Product, and Director of Product Development review the bugs which did not meet the 6-week delivery timeline. Align on next steps for which bugs to continue into the next sprint and which can be de-prioritized. | Mo Zhu |
+| Product confirm and celebrate                 | Weekly (Wednesday)       | The Head of Product meets with the designers and PMs to discuss completed epics. They also verify all updates to documentation, communications, guides, and the pricing and transparency pages, ensuring everything is set for the next steps.                                      | Mo Zhu            |
```

Comment:
Fleet does not use or refer to epics.  Only user stories are completed during a sprint.  "Epic" is a feature in Zenhub that allows for one (completely optional) way for teams to create sub-tasks: https://fleetdm.com/handbook/company/product-groups#sub-tasks

At Fleet, we don't use the word "epic", except in reference to the Zenhub feature, because it is confusing.